util/mongo.js

```diff
@@ -925,11 +984,20 @@ module.exports = {
             let result =  await _susp.findOne({ _id: memberId })
             return result.major.tier
         }
+        else if ( category == 'extreme' ) {
+            let member = await _susp.findOne({ _id: memberId } )
+            if ( member.extreme.tier < 1 ) return -1
+            await _susp.updateOne({ _id: memberId }, { $inc: { "extreme.tier": -1 }} )
+            let result =  await _susp.findOne({ _id: memberId })
+            return result.extreme.tier
+        }
     },
 
     isGoodyTwoShoes: async function ( memberId ) {
         let member = await _susp.findOne({ _id: memberId } )
         if ( !member || member === undefined ) return true
+        if ( member.extreme)
+            if (member.extreme.tier > 0 ) return false
```

Comment:
Something like this 
`
    let totalInfractionCount = 0;

    // Check and accumulate the counts for each infraction severity
    if (member.extreme && member.extreme.tier > 0) {
        totalInfractionCount += 1;
    }

    if (member.major && member.major.tier > 0) {
        totalInfractionCount += 1;
    }

    if (member.moderate && member.moderate.tier > 0) {
        totalInfractionCount += 1;
    }

    if (member.minor && member.minor.tier > 0) {
        totalInfractionCount += 1;
    }

    if (member.quit && member.quit.tier > 0) {
        totalInfractionCount += 1;
    }

    return totalInfractionCount === 0; // Return true if total count is zero, otherwise false`